lib/gax.js

```diff
@@ -58,6 +58,9 @@
  *   pageToken.
  * @property {boolean=} isBundling - If set to false and the call is configured
  *   for bundling, bundling is not performed.
+ * @property {number} maxRetries - the maximum number of retrying attempts that
```

Comment:
`_` prefix would have the similar sense in JS too, but I guess the community does not take this as seriously as Python community does. Explicitly saying the condition of the parameter in the documentation (like "please do not use this other than google-cloud-node project") might make more sense probably.

I have another idea here btw, which is: just removing the reference from the comments but keeping the code as-is.

This way, `google-cloud-node` can use it because they know it, but others can't find out this through the published documentation (if others look closer to the code, they'll find it out and could use it, but that would be probably acceptable).